src/Grpc.AspNetCore.Server/Internal/CallHandlers/ClientStreamingServerCallHandler.cs

```diff
@@ -57,8 +70,22 @@ protected override async Task HandleCallAsyncCore(HttpContext httpContext)
 
                 service = activator.Create();
 
-                response = await _invoker(
+                ClientStreamingServerMethod<TRequest, TResponse> resolvedInvoker = delegate (IAsyncStreamReader<TRequest> resolvedRequestStream, ServerCallContext resolvedContext)
```

Comment:
![image](https://user-images.githubusercontent.com/303201/56842818-095b6300-68ee-11e9-839c-6f3e00e7c26e.png)

Reply:
Even if there are no interceptors it looks like we're still wrapping the underlying method call. Can we make interceptors zero cost when they aren't present? (which should be most of the time)

Reply:
Yup, will need to optimize the fast path.